Gruntfile.js

```diff
@@ -29,16 +36,52 @@ module.exports = function (grunt) {
                 config: '.jscsrc'
             },
             src: '<%= config.files.lint %>'
-        }
+        },
```

Comment:
getting erros like ```
ReferenceError: module is not defined in /Users/minglee/Sites/djangular-rest-framework/tests/unit/test.js (line 10)
```
 not sure what the fix would be
looking at another of our repose i cant seem to find the defination.